src/common/components/select-repositories-page/index.js

```diff
@@ -52,6 +42,16 @@ class SelectRepositoriesPage extends Component {
       </div>
     );
   }
+
+  fetchData(props) {
```

Comment:
I wonder if it's possible to tell eslint to make sure custom methods come after render?